internal/nri/sandbox.go

```diff
@@ -39,6 +40,7 @@ func commonPodSandboxToNRI(pod PodSandbox) *nri.PodSandbox {
 		Annotations:    pod.GetAnnotations(),
 		RuntimeHandler: pod.GetRuntimeHandler(),
 		Pid:            pod.GetPid(),
+		Ips:            pod.IPs(),
```

Comment:
interestingly , crio already had a method to obtain this information

Reply:
let me know if you want to duplicate the method to be `GetIPs` for consistency with the other methods in the interface

Reply:
@aojea, would this be a lot of work? If not, then GetIPs() would be nice, indeed.

Reply:
It is not work at all, that was my preference in the beginning, but it duplicates an existing methods so I didn't know what what the preference of the maintainers

Reply:
done, I just wrapped the existin IPs() method instead of returning the field directly to ensure there can no be any drift in the future